src/ui/public/images/kibana.svg

```diff
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Adding the comment here since I can't comment on gifs, the `kibana.gif` above doesn't seem to be used anywhere